src/routes/volume/VolumeBulkActions.js

```diff
@@ -6,14 +6,47 @@ import style from './VolumeBulkActions.less'
 const confirm = Modal.confirm
 
 function bulkActions({ selectedRows, engineImages, bulkDeleteVolume, showBulkEngineUpgrade, showBulkChangeVolume, showBulkAttachHost, bulkDetach, bulkBackup, bulkExpandVolume, createPVAndPVC, createSchedule, confirmDetachWithWorkload, commandKeyDown }) {
+  const deleteWranElement = (rows) => {
+    let workloadResources = []
+    let pvResources = []
+    let hasVolumeAttached = false
+
+    rows.forEach((item) => {
+      if (item && item.kubernetesStatus && item.kubernetesStatus.pvStatus && item.kubernetesStatus.pvName) {
+        pvResources.push((<div style={{ margin: '10px', color: '#b7b7b7' }}>
+          <div style={{ paddingLeft: '10px' }}>PV Name: {item.kubernetesStatus.pvName}</div>
+          { !item.kubernetesStatus.lastPVCRefAt && item.kubernetesStatus.pvcName ? <div style={{ paddingLeft: '10px' }}>PVC Name: {item.kubernetesStatus.pvcName}</div> : ''}
+        </div>))
+      }
+      if (item.kubernetesStatus && item.kubernetesStatus.workloadsStatus && item.kubernetesStatus.workloadsStatus.length > 0) {
+        workloadResources.push((<div>
+          <div style={{ margin: '10px 0px', color: '#b7b7b7' }}>{item.kubernetesStatus.workloadsStatus.map((ele, i) => {
+            return (<div key={i} style={{ paddingLeft: '10px' }}>Pod Name: {ele.podName} ({ele.podStatus})</div>)
+          })}</div>
+        </div>))
+      }
+      if (item.state === 'attached') {
+        hasVolumeAttached = true
+      }
+    })
+    return (<div>
+      {hasVolumeAttached ? <div>Some of the selected volumes are attached to a node and may cause any applications using them once they are deleted!</div> : <div>Any volumes currently attached to a node may cause errors in any applications using them once they are deleted!</div>}
```

Comment:
I made a typo here. This should be:
```
Some of the selected volumes are attached to a node and may cause errors in any applications using them once they are deleted.
```
The other message starting with `Any volumes...` shouldn't be needed, even if none of the `Volumes` are attached.

@yasker Would it be good to list off the `Volumes` here that are attached for the user to easily see?

Reply:
When the volume is not attached, but there is a workload associated with it. without `with Any volumes...` the prompt message will be very strange

Reply:
Prompt `The following applications using these volumes:` when the volume is not atteched and there is an associated workload. is this OK ? @ttpcodes

Reply:
If a workload like a `Pod` is using the `Volume`, then the `Volume` should be attached I believe. Right now when we detect associated workloads we only look for `Pods` and not other resources that may be using the `Volume`, like `Stateful Sets` and `Deployments` where the `Volume` could be defined but not currently in use. So I think the message I suggested should be fine.

Reply:
It's a good idea if we can list the volumes here.

Reply:
Can I understand it this way, when the volume is not attached, we do not need to display the workload related information. If the workload information does not have a title, the style will look strange. like this 
![Jietu20200711-080022](https://user-images.githubusercontent.com/27733391/87211763-a072d600-c34d-11ea-9c5a-82506e7444f7.jpg)
@ttpcodes